### PR TITLE
[fix](test)Format ipv6

### PR DESCRIPTION
**Pull request by echo-hhj**
### What problem does this PR solve?

https://github.com/apache/doris/issues/42695

Issue Number: close 42695

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [x] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/common/proc/FrontendsProcNode.java [hunk 1]
```diff
@@ -214,10 +214,27 @@ private static boolean isJoin(List<InetSocketAddress> allFeHosts, Frontend fe) {
...
+    private static String formatIp(String context) {
+        // Formatting rule 1: Replace ":0:" with "::"
+        while (context.contains(":0:")) {
+            context = context.replaceAll(":0:", "::");
+        }
+
+        // Formatting rule 2: Replace ":::" with "::"
+        while (context.contains(":::")) {
+            context = context.replaceAll(":::", "::");
+        }
```
Comment (superdiaodiao) [inline fe/fe-core/src/main/java/org/apache/doris/common/proc/FrontendsProcNode.java:236]: IPv6 address has two `:` at most.

**Comment by echo-hhj** [comment]
run buildall

**Comment by echo-hhj** [comment]
run buildall

**Review by yujun777 — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Pull request closed without merging**